### DISCUSSION TITLE
Meeting 2024-07-16

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#776] [General]
This is the agenda for today's meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-07-16)

### Announcements

* Foundation grants are now live, and WG members are eligible for project grants: https://foundation.rust-lang.org/grants/fellowships/
  * Project goals have their own funding category too, see https://github.com/rust-embedded/wg/issues/775 for next time

**Comment by adamgreig** [comment]
Rollup of points from last week:

* Could c-m-rt gain a way to detect if a suitable linker script has been provided?
* We should schedule a meeting to discuss https://github.com/rust-embedded/wg/issues/775

**Reply by adamgreig** [reply]
Schedule is goals discussion July 23rd and another issue triage day Aug 6

**Comment by jamesmunns** [comment]
Minor ask, I could use an `r+` on https://github.com/rust-embedded/embedded-hal/pull/617, and we might want to see if we can test minimum dep versions in CI to avoid this in the future

**Comment by BartMassey** [comment]
MB2 Discovery Book status

* 90% done with the triage work. This leaves 90% to go.
* Hoping to move ownership of https://github.com/BartMassey-upstream/discovery-mb2 to rust-embedded this week?
* Reworked much of ch 9 "magnetic compass" this week. Will need review soon

**Comment by jamesmunns** [comment]
Discussions for an all-hands next year:

There's some discussion about an all hands next year, likely in NL, to celebrate the 10y anniversary of Rust.

We're not certain of how many spots will be available, but it would be good to get a poll of how many @rust-embedded/all members would be interested and able to travel to NL in the May-ish timeline next year.

**Reply by BartMassey** [reply]
I would love to be in on the All-Hands.

**Reply by hdoordt** [reply]
I'd surely love to join! Though if it means someone else who is more active than me, I wouldn't want to get in the way :)

**Reply by adamgreig** [reply]
I'm up for it!

**Reply by therealprof** [reply]
Me three.

**Reply by adamhott** [reply]
I'd love to join, but understand if there aren't enough chairs.

**Reply by Dirbaio** [reply]
+1

**Reply by kalkyl** [reply]
+1
